packages/@okta/vuepress-site/guides/sign-into-mobile-app/sign-in-page/iOS/signin.md

```diff
@@ -1,17 +1,51 @@
-Login events can be triggered by `@IBAction`, `viewDidLoad`, or programmatically. To launch the event, first update your `AppDelegate` to include the following function to allow the redirect to occur:
+Before using the SDK you have to create a new object of OktaOidc class. You can instantiate OktaOidc w/o parameters that means that SDK will use Okta.plist for configuration values. Alternatively you can create OktaOidc instance with custom configuration.
```

Comment:
Before using the SDK, you have to create a new object in the `OktaOidc` class. You can instantiate `OktaOidc` without parameters. If you do that, the SDK uses `Okta.plist` for configuration values. Alternatively, you can create the `OktaOidc` instance with a custom configuration.